qa/pull-tester/rpc-tests.py

```diff
@@ -32,12 +24,18 @@
 import re
 
 # Parse arguments and pass through unrecognised args
-parser = argparse.ArgumentParser(add_help=False)
-parser.add_argument('--coverage', action='store_true')
-parser.add_argument('-extended', action='store_true')
-parser.add_argument('--help', '-h', '-?', action='store_true')
-parser.add_argument('--parallel', type=int, default=4)
-parser.add_argument('-win', action='store_true')
+parser = argparse.ArgumentParser(add_help=False,
+                                 usage='%(prog)s [rpc-test.py options] [script options] [scripts]',
+                                 description=__doc__,
+                                 epilog='''
+Help text and arguments for individual test script:''',
+                                 formatter_class=argparse.RawTextHelpFormatter)
+parser.add_argument('--coverage', action='store_true', help='generate a basic coverage report for the RPC interface')
+parser.add_argument('--extended', action='store_true', help='run the extended test suite in addition to the basic tests')
+parser.add_argument('--help', '-h', '-?', action='store_true', help='print help text and exit')
+parser.add_argument('--jobs', '-j', type=int, default=4, help='how many test scripts to run in parallel. Default=4.')
+parser.add_argument('--nozmq', action='store_true', help='do not run the zmq tests')
+parser.add_argument('--win', action='store_true', help='signal that this is running in a Windows environment and that we should run the tests')
```

Comment:
Since you're adding a dash to this argument maybe consider renaming it something comprehensible like --force. And changing help text to "Run tests even on platforms where they are disabled by default (e.g. windows)."

The "signal that this is running in a windows environment" just comes off as strange, since the script obviously knows what environment it is running in.